staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/marshal.go

```diff
@@ -134,3 +201,32 @@ func (s *JSON) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+
+func (s JSON) MarshalCBOR() ([]byte, error) {
```

Comment:
Do we have full test coverage for `JSON.{Marshall|Unmarshal}CBOR`?

Reply:
Also, do we have coverage for JSONSchemaPropsOrStringArray?

Reply:
Good catch, we don't have coverage for that type's JSON marshal/unmarshal methods either.